src/plugins/ResizeReorder/ReorderCell.js

```diff
@@ -363,6 +384,25 @@ ReorderCell.propTypes = {
    * ```
    */
   onColumnReorderEnd: PropTypes.func.isRequired,
+
+  /**
+   * HACK to make this plugin work in a React v19 environment by letting the client pass the `createRoot` function from `react-dom/client`.
+   *
+   * Example usage:
+   * ```
+   * import { createRoot } from 'react-dom/client';
+   *
+   * const reorderCell = (
+   *  <ReorderCell
+   *   __useReactRoot={createRoot}
+   * />
+   * ```
+   *
+   * See https://github.com/schrodinger/fixed-data-table-2/issues/743) for more information.
+   *
+   * @deprecated This'll be removed in future major version updates of FDT.
+   */
+  __useReactRoot: PropTypes.func,
```

Comment:
nit: maybe we just call this something very explicit like `__react19RootCreator` so it's clear we're asking you to just pass in the `createRoot` API directly